tsdb/engine/b1/b1.go

```diff
@@ -445,7 +447,7 @@ func (e *Engine) FlushPartition(partitionID uint8) error {
 			}
 
 			// Write point to bucket.
-			if err := b.Put(u64tob(uint64(timestamp)), data); err != nil {
+			if err := b.Put(timestamp, data); err != nil {
```

Comment:
I changed it because it was only ever used to convert from `[]byte` -> `int64` -> `[]byte`